buildconfig/pygame-stubs/surface.pyi

```diff
@@ -1,49 +1,32 @@
-from typing import Any, List, Optional, Sequence, Text, Tuple, Union, overload, Iterable
+from typing import Any, Iterable, Optional, Sequence, Text, Tuple, Union, overload
+
 from pygame.bufferproxy import BufferProxy
 from pygame.color import Color
-from pygame.rect import Rect
-
 from pygame.math import Vector2
+from pygame.rect import Rect
 
-_ColorInput = Union[
-    Color, str, List[int], Tuple[int, int, int], Tuple[int, int, int, int]
-]
+_ColorInput = Union[Color, str, Sequence[int], Tuple[int, int, int], Tuple[int, int, int, int]]
```

Comment:
I'm not sure this takes any sequence at the moment.

Reply:
Will revert to List